sdk/OoyalaSkinSDK/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -274,6 +275,16 @@ - (void) bridgeAdStartNotification:(NSNotification *)notification {
   }
 }
 
+- (void) bridgeAdTappedNotification:(NSNotification *)notification {
```

Comment:
huuuh.  much cooler than i was originally thinking.  I'll see how it looks either today or monday.  awesome!